flux/grafana-manifests/dashboards/kustomization.yaml

```diff
@@ -9,7 +9,5 @@ resources:
   - fluxcd-flux-cluster-stats.yaml
   - fluxcd-flux-control-plane.yaml
   - fluxcd-gitops-flux-application-deployments-dashboard.yaml
-  - folders.yaml
```

Comment:
you are changing from `folder: ` to `folderRef: ` in all dashboards, but removing the reference to the file that seems to create the folders